From 36d14213d54921292a676afc1bb2da37fe297962 Mon Sep 17 00:00:00 2001
From: Bhakthavatsala Raghavendra <braghave@codeaurora.org>
Date: Thu, 18 Apr 2019 13:12:48 -0700
Subject: [PATCH 05/10] Bluetooth: Support to know if there is High Def call

Support to know if there is High Def call

CRs-fixed: 2443069

Change-Id: I534ee615f5fa3c58a4501b132dbdc6863705ee1c
---
 binder/android/bluetooth/IBluetoothHeadsetPhone.aidl | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/binder/android/bluetooth/IBluetoothHeadsetPhone.aidl b/binder/android/bluetooth/IBluetoothHeadsetPhone.aidl
index 830978065..9a51d1d43 100644
--- a/binder/android/bluetooth/IBluetoothHeadsetPhone.aidl
+++ b/binder/android/bluetooth/IBluetoothHeadsetPhone.aidl
@@ -31,6 +31,8 @@ interface IBluetoothHeadsetPhone {
   String getSubscriberNumber();
   boolean listCurrentCalls();
   boolean queryPhoneState();
+  boolean isHighDefCallInProgress();
+
 
   // Internal for phone app to call
   void updateBtHandsfreeAfterRadioTechnologyChange();
-- 
2.28.0

